src/catalog.py

```diff
@@ -529,19 +553,20 @@ def package(self, package, version):
 
 
 class NetBSDCatalog(Catalog):
-    def __init__(self, old_path, new_path):
+    def __init__(self, old_path: None, new_path: str) -> None:
```

Comment:
Same as above. Probably in more places below?

Reply:
I wonder if adding`build_packages.py` to the files to check would catch things like this.

Reply:
By using `mypy --strict` I found a whole bunch of stuff that should be fixed. Stand by...